src/CatsAndPlaceAndString/StringTest.java

```diff
@@ -0,0 +1,38 @@
+package CatsAndPlaceAndString;
+
+public class StringTest {
+    public void stringGet(String s) {
+        System.out.println(s);                           // Задание А
+    }
+
+
+    public static void main(String[] args) {
+        StringTest s1 = new StringTest();
+        String stg1 = new String("Hello World");
+        System.out.println(stg1.length());
+
+        s1.stringGet(stg1);// Задание 1
+
+        System.out.println(stg1.charAt(10));// Задание Б
```

Comment:
предпочитайте общие решения частным
```suggestion
        System.out.println(stg1.charAt(stg1.length() - 1));// Задание Б
```